erpnext/stock/doctype/item/item.js

```diff
@@ -5,6 +5,14 @@ frappe.provide("erpnext.item");
 
 frappe.ui.form.on("Item", {
 	setup: function(frm) {
+		frm.make_methods = {
+			'Purchase Order': () => {
+				frappe.model.open_mapped_doc({
+					method: 'erpnext.stock.doctype.item.item.make_purchase_order_item',
+					frm: frm
+				})
```

Comment:
- Do not need to do a make a frappe call
- Use frappe.new_doc here

Reply:
if i am using new_doc here it's not fetching 'item' in 'purchase order'.